test/unused-errors/find_unused_errors.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Rather than add this script to multiple repos, could we have it in the `actions` repo and just call it in ci? It will mean that you wont be able to run locally that easy (would just need a copy paste from the actions repo), but I dont know if that's too important.

Reply:
Moved to actions repo.  Running script locally probably won't be needed as action's output will be straight-forward in case it fails